build.rb

```diff
@@ -135,8 +135,7 @@
   python_checkup
 ]
 
-# LOCALES = ["", "es"]  # TODO: add support for the es locale when the content is merged
-LOCALES = [""]  # english locale only
+LOCALES = ["", "es"]  # english + spanish locales
```

Comment:
Why don't we specify `"en"`?

Reply:
`"en"` is specified by `""` since the `README.md` files have no extension for english content